tests/cpp/target_test.cc

```diff
@@ -448,6 +449,21 @@ TEST(TargetCreation, LLVMCommandLineSaveRestore) {
   ASSERT_FALSE(info.MatchesGlobalState());
 }
 
+TEST(TargetCreation, DetectSystemTriple) {
+  Map<String, ObjectRef> config = {
+      {"kind", String("llvm")},
+  };
+
+  Target target = Target(config);
+  ICHECK_EQ(target->kind, TargetKind::Get("llvm").value());
+
+  Optional<String> mtriple = target->GetAttr<String>("mtriple");
+  auto pf = tvm::runtime::Registry::Get("target.llvm_get_system_triple");
+  ASSERT_TRUE(pf->defined());
```

Comment:
Apologies, I'm not too sure what happened to the final part of this test. In its current form, it gets `mtriple` then doesn't use it, that wasn't intentional 😅

I'd suggest updating the test to something like:
```suggestion
  auto pf = tvm::runtime::Registry::Get("target.llvm_get_system_triple");
  if (pf == nullptr) {
    GTEST_SKIP() << "LLVM is not available, skipping test";
  }

  Optional<String> mtriple = target->GetAttr<String>("mtriple");
  ASSERT_TRUE(mtriple.value() == String((*pf)()));
```
which is closer to the original intention of the test.

By skipping at this stage, rather than earlier in the test, we can test that a call to `Target(config)` doesn't cause a segfault when `USE_LLVM=OFF`.